benchmark/wildtime_benchmarks/example_pipelines/arxiv.yaml

```diff
@@ -15,7 +15,7 @@ training:
   dataloader_workers: 2
   use_previous_model: True
   initial_model: random
-  batch_size: 96
+  batch_size: 128
   optimizers:
```

Comment:
128 is the largest size that fits on our gpus. We set 128 as default to make it convenient for new people to use

Reply:
What would be the implications of using smaller `batch_sizes` like `96`?

Reply:
Usually it's a common practice to use as a large batch size as possible (until the gpu cannot accommodate it), to speed up training. It could, in some cases, come with a cost of losing accuracy in the end, but as we compare different policies and keep the same batch size, it's fair. (i.e. we don't care how high the absolute accuracy can be, but how higher the accuracy one policy can reach compared to another).